src/UnderlineNav2/UnderlineNav.tsx

```diff
@@ -203,25 +195,8 @@ export const UnderlineNav = forwardRef(
       return breakpoint
     }
 
-    const [selectedLink, setSelectedLink] = useState<RefObject<HTMLElement> | undefined>(undefined)
```

Comment:
We don't really need `selectedLink` anymore as we manage the selected state with `aria-current`